west.yml

```diff
@@ -98,7 +98,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: v1.5.1
+      revision: pull/605/head
```

Comment:
Please update SHA: `8a673e7b4cdbfa4ff103ebf317f8fe3935cec9f1`

Reply:
Thank you